sbol_utilities/package.py

```diff
@@ -94,26 +95,33 @@ def define_package(doc: sbol3.Document):
         # Namespace must match the hasNamespace value for all of its members
         package.namespace = package_namespace
 
+        # Combine all of the docs into one
+        doc = args[0]
+        for sbol_doc in args[1:]:
+            doc_objects = [o for o in sbol_doc.objects]
+            doc.add(doc_objects)
+        
         # Add the package to the document
         doc.add(package)
```

Comment:
On reflection, I don't think we actually need or want to add the package automatically. We can build the package object without adding it to anything, then add it later (or in a convenience wrapper function, like this pass-through adder in component.py: https://github.com/SynBioDex/SBOL-utilities/blob/3fa0f3a2bac2833f82c486a6cfb0d66660963ceb/sbol_utilities/component.py#L80